### ISSUE TITLE
Keyboard Cursor and Selected Text

### ISSUE THREAD

**Issue by farb0d** [oivva/boxy#78]
Hey there.

Compare to the font and background color of the theme, the keyboard cursor is barely recognizable.
And so to selected text in dark theme.
![screen](https://cloud.githubusercontent.com/assets/13926209/16313033/9989b90c-398c-11e6-82c5-da5a0e783564.PNG)

Hope you fix it and we have a better theme for ST.
Thank you for your great effort and theme.

**Comment by oivva** [comment]
I can make it yellow, would that be ok? :wink: 

P.S. This issue is very similar to #74.

**Comment by farb0d** [comment]
I'm kind of obsessed with colors :))
But I think more light gray would work

What about keyboard cursor? making it slightly bolder would be great!

**Comment by oivva** [comment]
> I'm kind of obsessed with colors :))
Oh, I agree with you :smile: 

> But I think more light gray would work
Ok, I'll make it lighter. It will be available in 3.0.0. Before that you can change it [manually](https://github.com/oivva/boxy/wiki/Settings#advanced)

> What about keyboard cursor? making it slightly bolder would be great!
Try to play with default Sublime Text settings. I personally use next ones:

```js
"caret_extra_bottom": 1,
"caret_extra_top": 1,
"caret_extra_width": 1,
```

**Comment by farb0d** [comment]
Can't wait to see the new version.
Caret hack worked perfectly.
Thanks, really!
